src/components/navigation/HaukiNavigation.tsx

```diff
@@ -1,11 +1,15 @@
 import React, { useState } from 'react';
-import { Navigation } from 'hds-react';
+import { useHistory } from 'react-router-dom';
+import { Navigation, Notification } from 'hds-react';
+import api from '../../common/utils/api/api';
 import { AuthContextProps, useAuth } from '../../auth/auth-context';
 import './HaukiNavigation.scss';
 
 export default function HaukiNavigation(): JSX.Element {
+  const [signOutError, setSignError] = useState<string | undefined>();
```

Comment:
Isn't it customary to use the same name in the variable and the set? i.e. signOutError, setSignOutError.

Reply:
Oh. You changed it already. :D

Reply:
Yes, I noticed it too 😁